ports/restbed/fix-cmake.patch

```diff
@@ -24,40 +21,29 @@ diff --git a/CMakeLists.txt b/CMakeLists.txt
  set_property( TARGET ${STATIC_LIBRARY_NAME} PROPERTY CXX_STANDARD 14 )
  set_property( TARGET ${STATIC_LIBRARY_NAME} PROPERTY CXX_STANDARD_REQUIRED ON )
  set_target_properties( ${STATIC_LIBRARY_NAME} PROPERTIES OUTPUT_NAME ${PROJECT_NAME} )
--	
++set ( LIBRARY_NAME "${STATIC_LIBRARY_NAME}")
 +else ( )
+ 	
  set( SHARED_LIBRARY_NAME "${PROJECT_NAME}-shared" )
  add_library( ${SHARED_LIBRARY_NAME} SHARED ${MANIFEST} )
- set_property( TARGET ${SHARED_LIBRARY_NAME} PROPERTY CXX_STANDARD 14 )
-@@ -93,13 +94,20 @@ else ( )
+@@ -93,8 +96,13 @@ else ( )
  	set_target_properties( ${SHARED_LIBRARY_NAME} PROPERTIES OUTPUT_NAME ${PROJECT_NAME} )
  endif ( )	
  set_target_properties( ${SHARED_LIBRARY_NAME} PROPERTIES SOVERSION ${PROJECT_VERSION_MAJOR} VERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR} )
++set ( LIBRARY_NAME "${SHARED_LIBRARY_NAME}")
 +endif ( )
  
  if ( BUILD_SSL )
-+if ( BUILD_SHARED_LIBS )
++    target_link_libraries( ${LIBRARY_NAME} LINK_PRIVATE OpenSSL::SSL)
++elseif ( 1 )
```

Comment:
```suggestion
```

Reply:
This line is intentional to catch all other cases.
It is a minimal patch, not the most beautiful code ;-)

Reply:
Ah, I missed the `else` further down. To be honest, I think I would leave the patch as it is. It is certainly not as concise as your suggestion but I would rather go for readability in this case than minimizing the patch.

Reply:
Fine for me.